scripts/test-package-files.mjs

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+/* eslint-disable no-unused-vars, no-undef */
```

Comment:
It seems like some of the functions in here aren't defined anywhere and this might have been never finished, so I just ignored them for now

Reply:
I think the code never got used. It might be better to just delete the file.